css/_popover.scss

```diff
@@ -45,3 +45,7 @@
     margin: -16px -24px;
     z-index: $popoverZ;
 }
+
+.excalidraw .popover {
```

Comment:
Why is this necessary?

Reply:
Because the very generic selector from above messes up the margins for the excalidraw swatches popover.

Reply:
👍